worker/backend/cni_network.go

```diff
@@ -203,6 +220,15 @@ func (n cniNetwork) SetupMounts(handle string) ([]specs.Mount, error) {
 	}, nil
 }
 
+func (n cniNetwork) generateResolvConfContents() []byte {
+	contents := ""
+	for _, n := range n.nameServers {
+		contents = contents + fmt.Sprintf("nameserver %s\n", n)
```

Comment:
Nit: `n.nameServers` will never be large enough for this to have a meaningful performance impact, but could you change this to use a [`strings.Builder`](https://play.golang.org/p/Dv96oDHG0tF) or [`bytes.Buffer`](https://play.golang.org/p/b48TYKuEmiA) instead?

Reply:
aah we end up going with plain `+` for simplicity :grimacing:  in theory, for `strings.Builder` and `bytes.Buffer` we should check the error too

abf820a2291102912aff2cfbd98d38d57e949ead